metisp/pymetis/src/pymetis/tests/test_edps.py

```diff
@@ -0,0 +1,57 @@
+"""
+This file is part of the METIS Pipeline.
+Copyright (C) 2024 European Southern Observatory
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+"""
+
+import re
+import pytest
+import os
+import subprocess
+
+
+@pytest.fixture(autouse=True, scope="module")
+def reset_edps():
+    def inner():
+        os.system("edps -shutdown")
+        os.system("rm -rf /tmp/EDPS_Data/*")
+
+    return inner
```

Comment:
I'm not so comfortable with having *any* `os.system("rm -rf` code in the repository.

Reply:
Sure, it's dangerous. Right now I cannot think of a better way of resetting the state though.

Reply:
Yesterday I fixed the bug that the EDPS won't start if you delete your `EDPS_Data` directory, and I created this test:

https://github.com/AstarVienna/edps/blob/4b77d742c8de65ebde80eedf8c9d0ba822ed172e/test/tests/repository_tests.py#L446

(Created a fork of the EDPS to highlight that.) There an entire temporary `EDPS_Data` directory is created, and deleted after the test. Perhaps we can do something similar. I don't fully understand what is going on, I just cargo-culted that test.

(Which, if my fix is accepted, would mean that you would not need that `/*` in this part of the code.)

Maybe we can keep the code as-is for now. But I just wanted to note here that I don't like it. Which means by extension now I'm also on the line if something ever goes wrong with this code..